doc/README.md

```diff
@@ -42,7 +42,8 @@ Plugins
 * [hot reload](plugins.md): Hot reload without reload service.
 * [key-auth](plugins/key-auth.md): User authentication based on Key Authentication.
 * [JWT-auth](plugins/jwt-auth.md): User authentication based on [JWT](https://jwt.io/) (JSON Web Tokens) Authentication.
-* [basic-auth](doc/plugins/basic-auth.md): User authentication based on Basic Authentication.
+* [basic-auth](plugins/basic-auth.md): User authentication based on Basic Authentication.
+* [wolf-rbac](plugins/wolf-rbac.md) User Authentication and Authorization based on *RBAC*.
```

Comment:
should be `User Authentication based on *RBAC*.`?

Reply:
@iGeeky we can fix this by new PR